[2024-12-05T10:45:14.668+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: FoReport_file_processing.etl_merge_FO manual__2024-12-05T10:44:57.160745+00:00 [queued]>
[2024-12-05T10:45:14.735+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: FoReport_file_processing.etl_merge_FO manual__2024-12-05T10:44:57.160745+00:00 [queued]>
[2024-12-05T10:45:14.737+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-12-05T10:45:14.804+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): etl_merge_FO> on 2024-12-05 10:44:57.160745+00:00
[2024-12-05T10:45:14.845+0000] {standard_task_runner.py:57} INFO - Started process 275 to run task
[2024-12-05T10:45:14.901+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'FoReport_file_processing', 'etl_merge_FO', 'manual__2024-12-05T10:44:57.160745+00:00', '--job-id', '224', '--raw', '--subdir', 'DAGS_FOLDER/FoReport_file_processing.py', '--cfg-path', '/tmp/tmpq0bfh0m5']
[2024-12-05T10:45:14.907+0000] {standard_task_runner.py:85} INFO - Job 224: Subtask etl_merge_FO
[2024-12-05T10:45:15.041+0000] {task_command.py:410} INFO - Running <TaskInstance: FoReport_file_processing.etl_merge_FO manual__2024-12-05T10:44:57.160745+00:00 [running]> on host 8fff7f1050bf
[2024-12-05T10:45:15.428+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='FoReport_file_processing' AIRFLOW_CTX_TASK_ID='etl_merge_FO' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T10:44:57.160745+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-05T10:44:57.160745+00:00'
[2024-12-05T10:45:23.694+0000] {logging_mixin.py:150} INFO - IH with Escort data merged and stored successfully.
[2024-12-05T10:45:23.740+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-12-05T10:45:23.769+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=FoReport_file_processing, task_id=etl_merge_FO, execution_date=20241205T104457, start_date=20241205T104514, end_date=20241205T104523
[2024-12-05T10:45:23.842+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-12-05T10:45:23.879+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
